### PR TITLE
Added several planes and pictures.

### PR DESCRIPTION
**Pull request by pmd5700**
## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] While waiting for someone to review your request, please help review [another open pull request](https://github.com/sdr-enthusiasts/plane-alert-db/pulls) to support the maintainers.

### CODE REVIEW

**Review by dziban303 — APPROVED** [review]

**Comment by dziban303** [comment]
looks good to me, @Phaeton can you have a look too?

File: plane-alert-db.csv [hunk 1]
```diff
@@ -13241,3 +13241,9 @@ AE4A1D,166148,USAF,Raytheon T-6B Texan II,TEX2,Mil,Training Wheels,Bicycle Made
...
 A2A720,N270GP,Troy Aikman,Bombardier Challenger 300,CL30,Civ,NFL,The Other Football,Americas Team,Don't you know who I am?,https://en.wikipedia.org/wiki/Troy_Aikman
+AE0E23,88-26021,United States Army,UH-60M Blackhawk,H60,Mil,Get To The Choppa,Military Transport,Blackhawk,Toy Soldiers,https://en.wikipedia.org/wiki/Sikorsky_UH-60_Black_Hawk
+AE0E61,97-26767,United States Army,UH-60M Blackhawk,H60,Mil,Get To The Choppa,Military Transport,Blackhawk,Toy Soldiers,https://en.wikipedia.org/wiki/Sikorsky_UH-60_Black_Hawk
+ADFD86,94-00259,United States Army,Swearingen C-26B Metro,SW4,Mil,Bizjet,Not a Bus,Cargo,Toy Soldiers,https://en.wikipedia.org/wiki/Fairchild_C-26_Metroliner
+AE212F,02-26967,United States Army,UH-60M Blackhawk,H60,Mil,Get To The Choppa,Military Transport,Blackhawk,Toy Soldiers,https://en.wikipedia.org/wiki/Sikorsky_UH-60_Black_Hawk
+A70E5B,N554AV,AbbVie,Gulfstream G600,GA6C,Civ,Bizjet,Biotech,Pharmaceuticals,As Seen on TV,https://en.wikipedia.org/wiki/AbbVie
+AE0EF9,83-23908,United States Army,UH-60M Blackhawk,H60,Mil,Get To The Choppa,Military Transport,Blackhawk,Toy Soldiers,https://en.wikipedia.org/wiki/Sikorsky_UH-60_Black_Hawk
```
Comment (Phaeton) [inline plane-alert-db.csv:13249]:
```suggestion
AE0EF9,83-23908,United States Army,UH-60L Blackhawk,H60,Mil,Get To The Choppa,Military Transport,Blackhawk,Toy Soldiers,https://en.wikipedia.org/wiki/Sikorsky_UH-60_Black_Hawk

```

**Review by Phaeton — CHANGES_REQUESTED** [review]

File: plane_images.csv [hunk 2]
```diff
@@ -12052,7 +12052,7 @@ ADFF98,,,,
 ADFF96,https://cdn.jetphotos.com/full/5/53101_1542589471.jpg,,,
 ADFF95,https://cdn.jetphotos.com/full/6/93235_1635255120.jpg,https://cdn.jetphotos.com/full/5/33742_1602714762.jpg,https://cdn.jetphotos.com/full/2/43101_1286695850.jpg,
 ADFF94,,,,
-ADFF8D,,,,
+ADFF8D,https://cdn.jetphotos.com/full/6/27712_1650348168.jpg,https://cdn.jetphotos.com/full/6/34377_1551773043.jpg,https://cdn.jetphotos.com/full/6/52030_1561302932.jpg,
```
[the remark this answers is not in the document]
Comment (pmd5700) [reply plane_images.csv:12055]: Good catch!

**Review by pmd5700 — COMMENTED** [review]
Incorporated changes from review. Thanks!

**Review by Phaeton — APPROVED** [review]

**Pull request merged**